Build fails with "Could not find :spotify-sdk:"

* **Cordova:** 8.0.0
* **Gradle**: 4.1
* **Android:** 8.0 SDK Platform 26
* **OS**: Mac 10.13.1

**Reproduce:**

```
cordova create testapp
cd testapp
cordova platform add android
cordova plugin add cordova-spotify
cordova build android
```


When building I am getting the following build error

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':app:debugCompileClasspath'.
Could not find :spotify-sdk:.
Searched in the following locations:
   file:/<home>/Library/Android/sdk/extras/m2repository//spotify-sdk//spotify-sdk-.pom
   file:/<home>/Library/Android/sdk/extras/m2repository//spotify-sdk//spotify-sdk-.aar
   file:/<home>/Library/Android/sdk/extras/google/m2repository//spotify-sdk//spotify-sdk-.pom
   file:/<home>/Library/Android/sdk/extras/google/m2repository//spotify-sdk//spotify-sdk-.aar
   file:/<home>/Library/Android/sdk/extras/android/m2repository//spotify-sdk//spotify-sdk-.pom
   file:/<home>/Library/Android/sdk/extras/android/m2repository//spotify-sdk//spotify-sdk-.aar
   https://jcenter.bintray.com//spotify-sdk//spotify-sdk-.pom
   https://jcenter.bintray.com//spotify-sdk//spotify-sdk-.aar
   https://maven.google.com//spotify-sdk//spotify-sdk-.pom
   https://maven.google.com//spotify-sdk//spotify-sdk-.aar
   https://repo1.maven.org/maven2//spotify-sdk//spotify-sdk-.pom
   https://repo1.maven.org/maven2//spotify-sdk//spotify-sdk-.aar
   file:/<project>/platforms/android/app/libs/spotify-sdk-.aar
   file:/<project>/platforms/android/app/libs/spotify-sdk.aar
Required by:
   project :app
```

I'm not sure if it is a problem with Cordova or with this cordova-spotify.  I know the Spotify Android SDK docs says that the project is in beta and that things may be changing. I don't know if this problem has something to do with a recent change or not. Or maybe a problem with the paths (I noticed double slashes - not sure if that's correct or not).

--- a/src/android/build.gradle
+++ b/src/android/build.gradle
@@ -1,7 +1,7 @@
 repositories {
     jcenter()
     flatDir {
-        dirs 'libs'
+        dirs 'libs', 'src/main/libs'
     }
 }
 

